README.md

```diff
@@ -1260,7 +1260,7 @@ Other Style Guides
 
     > Why? They don't transpile well to ES5.
 
-  <a name="generators--spacing"></a>
+  <a name="generators--spacing"></a><a name="11.3"></a>
```

Comment:
please do not add any new links that have numbers. that's the old-style legacy format, and we won't be adding or changing numbered links forevermore.